coral-dbt/src/main/resources/tests/test_incremental.py

```diff
@@ -0,0 +1,78 @@
+#
+# Copyright 2023 LinkedIn Corporation. All rights reserved.
+# Licensed under the BSD-2 Clause license.
+# See LICENSE in the project root for license information.
+#
+import subprocess
+import unittest
+from jinja2 import Template
+
+SEEDS_DIR = 'seeds/'
+URL = 'http://localhost:8080/api/incremental/rewrite'
+
+class TestUtils:
+    @staticmethod
+    def construct_generate_incremental_code_test(name):
```

Comment:
can you rename these to introduce more clarity? 

something like `construct_expected_incremental_scala_script` & `construct_actual_incremental_scala_script`?